instrumentation/kamon-system-metrics/src/main/scala/kamon/instrumentation/system/host/HostMetrics.scala

```diff
@@ -4,127 +4,145 @@ import kamon.Kamon
 import kamon.instrumentation.system.host.HostMetrics.StorageDeviceInstruments.DeviceInstruments
 import kamon.instrumentation.system.host.HostMetrics.StorageMountInstruments.MountInstruments
 import kamon.instrumentation.system.host.HostMetrics.NetworkActivityInstruments.InterfaceInstruments
-import kamon.metric.{Counter, Gauge, InstrumentGroup, MeasurementUnit}
+import kamon.metric.{Counter, Gauge, Histogram, InstrumentGroup, MeasurementUnit}
 import kamon.tag.TagSet
 
 import scala.collection.mutable
 
 object HostMetrics {
 
-  val CpuUsage = Kamon.histogram (
+  val CpuUsage = Kamon.histogram(
     name = "host.cpu.usage",
     description = "Samples the CPU usage percentage",
     unit = MeasurementUnit.percentage
   )
 
-  val MemoryUsed = Kamon.gauge (
+  val MemoryUsed = Kamon.gauge(
     name = "host.memory.used",
     description = "Tracks the amount of used memory",
     unit = MeasurementUnit.information.bytes
   )
 
-  val MemoryFree = Kamon.gauge (
+  val MemoryUsage = Kamon.histogram(
```

Comment:
This metric should be a gauge, same as the `host.memory.used|free|total` metrics.